controllers/mirrorpeer_controller.go

```diff
@@ -725,8 +726,37 @@ func (r *MirrorPeerReconciler) deleteSecrets(ctx context.Context, mirrorPeer mul
 func (r *MirrorPeerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	r.Logger.Info("Setting up controller for MirrorPeer")
 
+	cmToMirrorPeerMapFunc := func(ctx context.Context, object client.Object) []ctrl.Request {
+		r.Logger.Debug("Mapping ConfigMap to MirrorPeer", "ConfigMap", client.ObjectKeyFromObject(object))
+		var reqs []ctrl.Request
+		cm, ok := object.(*corev1.ConfigMap)
+		if !ok || cm.Name != utils.ClientInfoConfigMapName {
+			r.Logger.Debug("Got unexpected ConfigMap. Not requeing any requests.")
+			return reqs
+		}
```

Comment:
```suggestion
		if !ok {
			r.Logger.Debug("Unable to cast object into a ConfigMap. Not requeing any requests.")
			return reqs
		}
```

You're already checking if name matches with the predicates below

Reply:
I just wanted to be double sure.